go/teams/service_helper.go

```diff
@@ -762,6 +762,22 @@ func EditMemberByID(ctx context.Context, g *libkb.GlobalContext, teamID keybase1
 	return editMember(ctx, g, teamGetter, username, role, botSettings)
 }
 
+func EditMembers(ctx context.Context, g *libkb.GlobalContext, teamname string, users []keybase1.UserRolePair) (res keybase1.TeamAddMembersResult, err error) {
+	var notAdded []keybase1.User
+
+	for _, userRolePair := range users {
+		err := EditMember(ctx, g, teamname, userRolePair.AssertionOrEmail, userRolePair.Role, userRolePair.BotSettings)
+		if err != nil {
+			user := keybase1.User{Username: libkb.NewNormalizedUsername(userRolePair.AssertionOrEmail).String()}
```

Comment:
This is fishy. `AssertionOrEmail` can be `xyz@twitter.com` which is not a valid `NormalizedUsername`.

In `teams.AddMembers` `notAdded` is only used for contact restrictions.

I think it would make sense for `TeamEditMembers` to return something like this instead.
```
  record TeamAddMembersResult {
    array<UserRolePair> failures;
  }
```

Reply:
Yeah that's very 🐟 , will make those changes